app/styles/organisms/sidebar.less

```diff
@@ -0,0 +1,128 @@
+@media screen and (-webkit-device-pixel-ratio: 1) {
+  .oss-sidebar-item--locked {
+    padding-left: 1px;
+    padding-bottom: 1px;
+  }
+
+  .oss-sidebar-item--icon {
+    padding: 1px;
+  }
+}
+
+.oss-sidebar--containers {
+  position: fixed;
+  height: 100vh;
+  width: var(--sidebar-width);
+
+  background-color: var(--sidebar-bg-color);
+  border-right: 1px solid var(--sidebar-outer-border-color);
+  padding: var(--spacing-px-12) var(--spacing-px-9);
+
+  .logo-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-height: var(--topbar-height);
+    max-height: var(--topbar-height);
+
+    margin-right: -1px;
+    border-bottom: 1px solid var(--sidebar-inner-border-color);
+    img {
+      width: 30px;
+      height: 30px;
+    }
+  }
+
+  .oss-sidebar--content {
+    padding: var(--spacing-px-12) 0px;
+  }
+
+  .oss-sidebar--footer {
+    border-top: 1px solid var(--sidebar-inner-border-color);
+    margin-right: -1px;
+    padding: var(--spacing-px-12) var(--spacing-px-9);
+  }
+}
+
+.oss-sidebar-item {
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 42px;
+  width: 42px;
+
+  color: var(--sidebar-font-color);
+  font-size: var(--font-size-lg);
+  border-radius: var(--border-radius-md);
+
+  cursor: pointer;
+
+  &:hover {
+    text-decoration: none;
+    color: var(--sidebar-font-color);
+  }
+
+  &:not(:hover):not(.active) .oss-sidebar-item--icon,
```

Comment:
From what I have tested locally, Locked items don't have a hover state, that's normal right ?

Reply:
This has been seen with William